src/style.css

```
@@ -0,0 +1,71 @@
+* {
+  margin: 0;
+}
+
+.body {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.body_item {
+  display: flex;
+  flex-grow: 1;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  font-family: Arial, Helvetica, sans-serif;
+  color: white;
+  min-width: 300px;
+  height: 300px;
+  background-color: rgb(255,0,0);
+  font-size: 100px;
+}
+
+.body_item:nth-child(2) {
+  background-color: rgb(204,0,0);
+}
+
+.body_item:nth-child(3) {
+  background-color: rgb(153,0,0);
+}
+
+.body_item:nth-child(4) {
+  background-color: rgb(102,0,0);
+}
+
+.body_item:nth-child(5) {
+  background-color: rgb(51,0,0);
+}
+
+.body_item:nth-child(6) {
+  background-color: rgb(0,0,0);
+}
+
+@media screen and (min-width: 900px) {
+  .body_item {
+    width: 30%;
+  }
+  .body_item:nth-child(6) {
+    order: 0;
+  }
+  .body_item:nth-child(4) {
+    order: 2;
+  }
+  .body_item:nth-child(5) {
+    order: 1;
+  }
+}
+
+@media screen and (min-width: 600px) and (max-width: 899px) {
+  .body_item {
+    width: 50%;
+  }
+  .body_item:nth-child(3) {
+    order: 1;
+  }
+  .body_item:nth-child(5),
+  .body_item:nth-child(6) {
+    order: 2;
+  }
+}
```

```suggestion

@media screen and (min-width: 600px) {
  .body_item {
    width: 50%;
  }
  .body_item:nth-child(3) {
    order: 1;
  }
  .body_item:nth-child(5),
  .body_item:nth-child(6) {
    order: 2;
  }
}

@media screen and (min-width: 900px) {
  .body_item {
    width: 30%;
  }
  .body_item:nth-child(6) {
    order: 0;
  }
  .body_item:nth-child(4) {
    order: 2;
  }
  .body_item:nth-child(5) {
    order: 1;
  }
}
```
Если записать так - `and` можно не писать